Gem/Code/CMakeLists.txt

```diff
@@ -11,6 +11,7 @@ ly_add_target(
     NAME MultiplayerSample.Client.Static STATIC
     NAMESPACE Gem
     FILES_CMAKE
+        multiplayersample_client_files.cmake
```

Comment:
Nit: make order consistent (I would use ordering from below)

Reply:
Thank you, fixed!